docs/book/v6/tutorials/create-book-module.md

```diff
@@ -88,47 +135,99 @@ class ConfigProvider
     {
         return [
             'delegators' => [
-                Application::class => [
-                    RoutesDelegator::class
-                ]
+                Application::class              => [RoutesDelegator::class],
+                PostBookResourceHandler::class          => [HandlerDelegatorFactory::class],
+                GetBookResourceHandler::class           => [HandlerDelegatorFactory::class],
+                GetBookCollectionHandler::class => [HandlerDelegatorFactory::class],
             ],
-            'factories' => [
+            'factories'  => [
+                PostBookResourceHandler::class          => AttributedServiceFactory::class,
+                GetBookResourceHandler::class           => AttributedServiceFactory::class,
+                GetBookCollectionHandler::class => AttributedServiceFactory::class,
+                BookService::class              => AttributedServiceFactory::class,
             ],
-            'aliases'   => [
+            'aliases'    => [
+                BookServiceInterface::class => BookService::class,
             ],
         ];
     }
 
-    private function getDoctrineConfig(): array
+    private function getHalConfig(): array
     {
         return [
-            
+            AppConfigProvider::getResource(Book::class, 'book::view-book'),
+            AppConfigProvider::getCollection(BookCollection::class, 'book::list-books', 'books'),
         ];
     }
+}
+```
 
-    private function getHalConfig(): array
+* `src/Core/src/Book/src/ConfigProvider.php`
+
+In `src/Core/src/Book/src` we will create 1 PHP file: `ConfigProvider.php`. This file contains the necessary configuration for Doctrine ORM.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace Core\Book;
+
+use Core\Book\Repository\BookRepository;
+use Doctrine\ORM\Mapping\Driver\AttributeDriver;
+use Dot\DependencyInjection\Factory\AttributedRepositoryFactory;
+
+class ConfigProvider
+{
+    public function __invoke(): array
+    {
+        return [
+            'dependencies'     => $this->getDependencies(),
+            'doctrine'         => $this->getDoctrineConfig(),
+        ];
+    }
+
+    private function getDependencies(): array
     {
         return [
-           
+            'factories' => [
+                BookRepository::class => AttributedRepositoryFactory::class,
+            ],
         ];
     }
 
+    private function getDoctrineConfig(): array
+    {
+        return [
+            'driver' => [
+                'orm_default'  => [
+                    'drivers' => [
+                        'Core\Book\Entity' => 'BookEntities',
+                    ],
+                ],
+                'BookEntities' => [
+                    'class' => AttributeDriver::class,
+                    'cache' => 'array',
+                    'paths' => [__DIR__ . '/Entity'],
+                ],
+            ],
+        ];
+    }
 }
 ```
 
 ### Registering the module
```

Comment:
This section should also be moved further down to the point where the `ConfigProvider` has already been created.